test/journal/create_new_entry_test.js

```diff
@@ -85,6 +85,37 @@ describe("Journal", function () {
             return expect(createPatientEntry({ mood: "Happy!" })).to.be.a.journal.createSuccess;
         });
 
+        //moodEmoji
+        it("doesn't require a mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: undefined })).to.be.a.journal.createSuccess;
+        });
+        it("rejects a blank mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: "" })).to.be.an.api.error(400, "invalid_emoji");
+        });
+        it("allows a null mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: null })).to.be.a.journal.createSuccess;
+        });
+        it("allows a unicode mood Emoji", function () {
+            return createPatientEntry({moodEmoji : "\\U0001F625"})
+            .then(function (response){
+                expect(response).to.be.a.journal.createSuccess;
+                expect(response.body.moodEmoji).to.match(/^\\U[\da-f]{1,8}$/i);
+            });
+        });
+        it("does not allow mood Emoji non-unicode string", function (){
+            return createPatientEntry({moodEmoji : "abcd12345"})
+            .then(function (response){
+                expect(response).to.be.an.api.error(400, "invalid_emoji");
+                expect(response.body.moodEmoji).to.not.match(/^\\U[\da-f]{1,8}$/i);
```

Comment:
API errors don't include the record data in their response (because the error essentially means the API is unsure what the record data should be), so there isn't a `response.body.moodEmoji` field. So you don't need line 109, and can actually reformat the whole black into `return expect(createPatientEntry({moodEmoji : "abcd12345"})).to.be.an.api.error(400, "invalid_emoji");`